packages/storefront-lwc/src/modules/commerce/api/client.js

```diff
@@ -18,7 +18,7 @@ const authLink = new ApolloLink((operation, forward) => {
 const defaultOptions = {
     watchQuery: {
         fetchPolicy: 'no-cache',
-        errorPolicy: 'ignore',
+        errorPolicy: 'all',
```

Comment:
`ignore` is a bad policy for us, it essentially means whenever there is an `ApolloError` it is invisible to the client